test/helper/webapi.js

```diff
@@ -648,7 +648,8 @@ module.exports.tests = function () {
     });
   });
 
-  describe('#attachFile', () => {
+  describe.skip('#attachFile', () => {
+    // Those tests are skipped due to bug https://github.com/puppeteer/puppeteer/issues/5543
```

Comment:
could you mark this as ` // TODO: ....` ?

possibly, we can create issue with this problem, to see it
and type link on it `// TODO: #xxxx bla bla....` or `// TODO: https://github.com/Codeception/CodeceptJS/issues/xxxx bla bla...`